vnext/Microsoft.ReactNative.Cxx.UnitTests/ReactModuleBuilderMock.h

```diff
@@ -120,6 +120,14 @@ struct ReactModuleBuilderMock {
 struct ReactContextMock : implements<ReactContextMock, IReactContext> {
   ReactContextMock(ReactModuleBuilderMock *builderMock) noexcept;
 
+  ReactPropertyBag GlobalProperties() noexcept {
+    VerifyElseCrashSz(false, "Not implemented");
```

Comment:
nit: Does the motif adapter eat crashes? If not it may be better to fall the test instead of crashing to get better output.

Reply:
This is a mock implementation. Our test does not use it. We use VEC mostly to make sure that no one calls it in the test. If they do then we crash and must see it. The real code is exercised in the Microsoft.ReactNative.IntegrationTests

Reply:
Fully understand it's  a mock and those methods aren't called. What I mean is that if they ever do end up getting called, failing the test would be a bit less cumbersome than crashing to figure out what's going on.

Reply:
To answer your question: no, the motif adapter does not eat crashes unless someone use an explicit test macro TestCheckCrash. We even have TestCheckTerminate macro that checks that std::terminate is called. We use setjmp/longjmp to survive the the app from the termination. Both this macros only wrap up the code passed to them. They do not affect any other common unit test functionality.

Reply:
Got it. I know within Office it will eat crashes and fail the test with a message saying the test crashed. Without eating crashes, it seems like crashing would lead to confusion in results unless you attached a debugger.

E.g. imagine seeing a CI failure saying "xxx method was called" vs seeing that crash happened during UTs.

Reply:
Not a particularly big deal.